templates/partials/cd-header.tpl

```diff
@@ -136,9 +137,10 @@
                 <span class="cd-menu__profile-name"></span>
               </div>
               <ul class="cd-menu__content-block cd-menu__profile-menu">
-                <li><a class="cd-menu__profile-link">My Profile</a></li>
-                <li><a href="/dashboard/my-dojos">My Dojos</a></li>
-                <li><a href="/dashboard/dojos/events/user-events">My Events</a></li>
+                <li><a class="cd-menu__profile-link">Forum Profile</a></li>
```

Comment:
So, I know this was broken before, but this is missing the href for actually going to the Forum Profile.